src/api/v3/imprint.py

```diff
@@ -58,5 +60,10 @@ def imprint(request, region_slug, language_code):
         imprint_translation = region.imprint.get_public_translation(language_code)
         if imprint_translation:
             return JsonResponse(transform_imprint(imprint_translation))
+        imprint_default_translation = region.imprint.get_public_translation(
+            region.default_language.code
```

Comment:
If we want to be 100% sure to not throw an error, we should check whether `region.default_language` exists before accessing the `code` field. But this only affects an edge case where a new region is created without any languages...

Reply:
@Wiedemsa do you want to give it a try? :-)

Reply:
i will check it out :)